elastica/callback_functions.py

```diff
@@ -111,52 +113,59 @@ class ExportCallBack(CallBackBaseClass):
     def __init__(
         self,
         step_skip: int,
-        path: str,
+        filename: str,
+        directory: str,
         method: str,
         initial_file_count: int = 0,
         file_save_interval: int = 1e8,
     ):
         """
-
         Parameters
         ----------
         step_skip : int
             Interval to collect simulation data into buffer.
             The data will be collected at every `dt * step_skip`
             interval.
-        path : str
-            Path to save the file. If directories are prepended,
-            they must exist. The filename depends on the method.
-            The path is not expected to include extension.
+        filename : str
+            Name of the file without extension. The extension will be
+            determined depend on the method.
+        directory : str
+            Path to save the file. If directory doesn't exist, it will
+            be created. Any existing files will be overwritten.
```

Comment:
This should probably say:

_Directory to save the file. If directory doesn't exist, it will be created. Any existing files in the directory will be overwritten._

Reply:
Okay, I'll clarify.

Reply:
[9e16e3d](https://github.com/GazzolaLab/PyElastica/commit/9e16e3d484d9c30673cc00897e5d06b091db08c4)